Libraries/Components/DatePicker/DatePickerIOS.ios.js

```diff
@@ -118,18 +128,24 @@ var DatePickerIOS = React.createClass({
 
   render: function() {
     var props = this.props;
+    let minimumDate = props.minimumDate;
+    let maximumDate = props.maximumDate;
+
+    if (typeof minimumDate === 'object') {
+      minimumDate = minimumDate.getTime();
+    }
+    if (typeof maximumDate === 'object') {
+      maximumDate = maximumDate.getTime();
+    }
+
     return (
       <View style={props.style}>
         <RCTDatePickerIOS
           ref={ picker => this._picker = picker }
           style={styles.datePickerIOS}
           date={props.date.getTime()}
```

Comment:
Tests are failing here: 
https://travis-ci.org/facebook/react-native/jobs/123110901#L1531

I think you need to pass`propsTimeStamp` as the `date`prop.